Formula/apache-arrow.rb

```diff
@@ -4,6 +4,7 @@ class ApacheArrow < Formula
   url "https://www.apache.org/dyn/closer.cgi?path=arrow/arrow-0.15.0/apache-arrow-0.15.0.tar.gz"
   sha256 "d1072d8c4bf9166949f4b722a89350a88b7c8912f51642a5d52283448acdfd58"
   head "https://github.com/apache/arrow.git"
+  revision 1
```

Comment:
According to `brew audit apache-arrow`, this line should come before the `head` line.

Reply:
Changed the revision line